triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -446,18 +450,28 @@ trait ToxiSandboxFixture extends BeforeAndAfterAll with ToxiproxyFixture with Sa
   }
 }
 
+// TODO: delete once Oracle and Postgres migrations are completed
 trait AbstractTriggerDaoFixture extends SuiteMixin {
   self: Suite =>
 
   protected def jdbcConfig: Option[JdbcConfig]
 }
 
-trait TriggerDaoInMemFixture extends AbstractTriggerDaoFixture {
+// TODO: rename once Oracle and Postgres migrations are completed
+trait AbstractTriggerDaoCantonFixture extends CantonFixture {
```

Comment:
Test migration strategy is to:
- copy and paste existing testing and add a `WithCanton` or similar prefix
- migrate the `WithCanton` tests to use `CantonFixture`